source/frontend/src/features/contacts/contact/create/Organization/CreateOrganizationForm.tsx

```diff
@@ -39,15 +35,15 @@ export const CreateOrganizationForm: React.FunctionComponent<unknown> = () => {
   // validation needs to be adjusted when country == OTHER
   const { otherCountryId } = useAddressHelpers();
 
-  const formikRef = useRef<FormikProps<IEditableOrganizationForm>>(null);
+  const formikRef = useRef<formik.FormikProps<IEditableOrganizationForm>>(null);
 
   const onSubmit = async (
     formOrganization: IEditableOrganizationForm,
-    { setSubmitting }: FormikHelpers<IEditableOrganizationForm>,
+    { setSubmitting }: formik.FormikHelpers<IEditableOrganizationForm>,
```

Comment:
can we try to keep our PRs a bit more focused, I don't disagree with this change but it has nothing to do with removing old models, and this change also hasn't been applied across the project. When we come across something like this my preference would be to create a separate PR, and move the whole code base over, and also make sure the team knows. Otherwise we are just going to keep editing files back and forth.

Reply:
I agree, I think i was debugging this file and made the change. I can undo it to lower the things change that are not related to the pr